.github/workflows/tests.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Set up environment
-        uses: conda-incubator/setup-miniconda@v3.0.4
```

Comment:
This is the relevant change, the rest is just pinning consistently to the minor version.

Reply:
I have overestimated the capabilities of this action. Consistently pinning to v3 now.